tests/models/sniff_test.py

```diff
@@ -0,0 +1,84 @@
+# pylint: disable=line-too-long,bad-whitespace,no-self-use
```

Comment:
Not super thrilled with the long lines or disabling bad whitespace.  I assume you just copy-pasted from the demo, or something?

Reply:
I can fix the whitespace, the long lines will be really hard, they're mostly just really long strings. I would have to really go out of my way to get them under 115